apps/promotion-app/webapp/src/view/admin/panel/userManagement.tsx

```
@@ -0,0 +1,1153 @@
+        dialogContext.showConfirmation(
+            "Confirm Acceptance",
+            `Are you sure you want to make thsi transfer?`,
+            ConfirmationType.accept,
+            async () => {
+                const resultAction = await dispatch(updateUser({
+                    id: user.id,
+                    email: toLeadEmail,
+                    roles: user.roles,
+                    functionalLeadAccessLevels:
+                        user.roles.includes(Role.FUNCTIONAL_LEAD) &&
+                        user.functionalLeadAccessLevels?.businessUnits.length || 0 > 0
```

Biome (2.4.15)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)